bugsnag-android-core/src/main/java/com/bugsnag/android/ImmutableConfig.kt

```diff
@@ -5,21 +5,21 @@ import java.util.HashMap
 
 internal data class ImmutableConfig(
     val apiKey: String,
-    val autoNotify: Boolean,
-    val detectAnrs: Boolean,
-    val detectNdkCrashes: Boolean,
-    val autoCaptureSessions: Boolean,
+    val autoDetectErrors: Boolean,
+    val autoDetectAnrs: Boolean,
+    val autoDetectNdkCrashes: Boolean,
+    val autoTrackSessions: Boolean,
     val autoCaptureBreadcrumbs: Boolean,
     val sendThreads: Boolean,
     val ignoreClasses: Collection<String>,
-    val notifyReleaseStages: Collection<String>,
+    val enabledReleaseStages: Collection<String>,
     val projectPackages: Collection<String>,
     val releaseStage: String?,
     val buildUuid: String?,
     val appVersion: String?,
     val versionCode: Int,
     val codeBundleId: String?,
-    val notifierType: String?,
+    val appType: String,
```

Comment:
This is now guaranteed to be non-null

Reply:
See my other comments – I don't think this should have a default value. Am fine with it being an empty string if that helps in avoiding nullness.